syntax = "proto3";

option java_multiple_files = true;

package demo.grpc;

message DateTimeMsg {
	string value = 1;
}

service DateTime{
	rpc getCurrentDateTime(DateTimeMsg) returns (DateTimeMsg) {}
}
